src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -37,15 +39,25 @@ public List<Resume> GetAll()
             List<Resume> resumes = new List<Resume>();
             using (IDbConnection db = new SqlConnection(this.connectionString))
             {
-                resumes = db.Query<Resume>("select * from resume").ToList();
+                resumes = db.Query<Resume>("select Id, Title, Sumary, CreationDate, LastUpdate from Resume").ToList();
             }
 
             return resumes;
         }
 
         public Resume GetById(Guid id)
         {
-            throw new NotImplementedException();
+            var sql = "select Id, Title, Sumary, CreationDate, LastUpdate from Resume where Id=@id";
+            Resume resume = new Resume();
+            using (IDbConnection db = new SqlConnection(this.connectionString))
+            {
+                db.Open();
+                DynamicParameters parameter = new DynamicParameters();
+                parameter.Add("@id", id);
+                db.Execute(sql, parameter);
```

Comment:
you are not storing this result in the resume object, so you are returning a null resume

Reply:
ups

Reply:
Done